apps/src/javalab/JavalabEditor.jsx

```diff
@@ -85,6 +93,51 @@ class JavalabEditor extends React.Component {
     });
   }
 
+  makeListeners(key) {
+    return {
+      onContextMenu: e => {
+        this.handleTabContextMenu(key, e);
+      }
+    };
+  }
+
+  handleTabContextMenu(key, e) {
+    e.preventDefault();
+    const boundingRect = e.target.getBoundingClientRect();
+    this.setState({
+      showMenu: true,
+      contextTarget: key,
+      menuPosition: {
+        top: `${boundingRect.bottom}px`,
+        left: `${boundingRect.left}px`
```

Comment:
How will this work for RTL languages? We don't necessarily need to figure this out now, but if it doesn't work by default, go ahead and make a bug on [this epic](https://codedotorg.atlassian.net/browse/CSA-258) and we'll address it later.